### ISSUE TITLE
doesn't work on 13.4.1

### ISSUE THREAD

**Issue by kasperkamperman** [kasperkamperman/MobileCameraTemplate#9]
Because of dependency on DetectRTC. 
See this open issue: 
https://github.com/muaz-khan/DetectRTC/issues/88

**Comment by kasperkamperman** [comment]
I tried to remove the DetectRTC dependency. That worked for iOS and I got a working camera in 13.4.1, however it broke the camera on my Android device...

```
// this function counts the amount of video inputs
// it replaces DetectRTC that was previously implemented
function deviceCount() {
  return new Promise(function (resolve) {
    if (
      navigator.mediaDevices &&
      navigator.mediaDevices.getUserMedia &&
      navigator.mediaDevices.enumerateDevices
    ) {
      var videoInCount = 0;

      navigator.mediaDevices
        .enumerateDevices()
        .then(function (devices) {
          devices.forEach(function (device) {
            if (device.kind === "video") {
              device.kind = "videoinput";
            }

            if (device.kind === "videoinput") {
              videoInCount++;
            }
          });

          resolve(videoInCount);
        })
        .catch(function (err) {
          console.log(err.name + ": " + err.message);
          resolve(0);
        });
    } else resolve(0);
  });
}

document.addEventListener("DOMContentLoaded", function (event) {
  deviceCount().then(function (deviceCount) {
    if (deviceCount > 0) {
      amountOfCameras = deviceCount;
      initCameraUI();
      initCameraStream();
    } else {
      alert(
        "Mobile camera is not supported by browser, or there is no camera detected/connected"
      );
    }
  });
});
```

If you want to play around with it and find out why this doesn't work on Android, be my guest.

**Comment by kasperkamperman** [comment]
I think I solved the 13.4.1 issue with a new update.

**Comment by David-dp-** [comment]
Which browser/UA uses "video" for a device kind? All the ones I've tested use "videoinput".
